pypopquiz/backends/ffmpeg.py

```diff
@@ -13,7 +13,7 @@
 class FFMpeg(ppq.backends.backend.Backend):
     """FFMPEG backend, implements interface from base-class"""
 
-    def __init__(self, source_file: Path, display_graph: bool = False) -> None:
+    def __init__(self, source_file: Path, display_graph: bool = False, width: int = 1280, height: int = 720) -> None:
```

Comment:
Since we have this here now, should we store this as a member variable and remove these arguments from `scale_video` and `draw_text_in_box` (from the abstract class and from the ffmpeg class as well)?

Reply:
Agreed, less arguments would be nice. I could edit the base class, give it a constructot that stores the width and height, and use it everywhere.